controllers/usersignup/usersignup_controller_test.go

```diff
@@ -4669,6 +4669,33 @@ func TestUserReactivatingWhileOldSpaceExists(t *testing.T) {
 	})
 }
 
+func TestHomeSpace(t *testing.T) {
```

Comment:
I would expect that there are already existing tests which covers space creation. Can we just update those tests to make sure that the home space is set in the status instead of creating a new test for this case?

Reply:
Done. fa1cb589eb086f3e637b32baecd03e16b1231a0c